package-system/AWSGameLiftServerSDK/README.md

```diff
@@ -0,0 +1,117 @@
+# GameLiftServerSdk C++
+## Documentation
+You can find the official GameLift documentation [here](https://aws.amazon.com/documentation/gamelift/).
+
+## Minimum requirements:
+* Either of the following:  
+  * Microsoft Visual Studio 2017 - Windows Build
+  * Clang 6.0 - Linux Build
+* CMake version 3.1 or later
+* Python version 3.6 or later
+* A Git client available on the PATH
+
+## Build the GameLiftServerSdk - Windows
+1. Download the latest [GameLift Server SDK](https://aws.amazon.com/gamelift/getting-started/) 
+   and unzip source `GameLift-Cpp-ServerSDK-{version-number}`
+   
+2. Copy and put `package-system/AWSGameLiftServerSDK/build_and_package.py` and `package-system/AWSGameLiftServerSDK/gamelift-sdk.json` 
+   into `GameLift-Cpp-ServerSDK-{version-number}` directory in step 1
+   
+3. Run the python script `build_and_package.py` under `GameLift-Cpp-ServerSDK-{version-number}` directory, like
+   ```
+   & python build_and_package.py --config gamelift-sdk.json --platform windows --version 3.4.1
+   ```
+   
+4. Build package will be located under `GameLift-Cpp-ServerSDK-{version-number}/build/{version-number}` directory
+
+5. Copy and put build package into `package-system/AWSGameLiftServerSDK/windows/AWSGameLiftServerSDK`, for example
+   ``` 
+   - AWSGameLiftServerSDK
+     - bin
+     - include
+     - lib
+     - LICENSE_AMAZON_GAMELIFT_SDK.txt
+     - NOTICE_C++_AMAZON_GAMELIFT_SDK.txt
+   ```
+6. Update `package-system/AWSGameLiftServerSDK/windows/PackageInfo.json` with built version `PackageName`
```

Comment:
Sorry but I'm a little confused about this line: Do we expect users to update the build version or package name in PackageInfo.json?

Reply:
yes, we expect operator to update package name based on the version they build.